### PR TITLE
Lower the version requirement of cmake from 3.12 to 3.2

### PR DESCRIPTION
**Pull request by anwu1219**

### CODE REVIEW

File: src/configuration/GlobalConfiguration.cpp [hunk 1]
```diff
@@ -45,12 +45,12 @@ const double GlobalConfiguration::COST_FUNCTION_ERROR_THRESHOLD = 0.0000000001;
 
 const bool GlobalConfiguration::USE_HARRIS_RATIO_TEST = true;
 
-const bool GlobalConfiguration::USE_SYMBOLIC_BOUND_TIGHTENING = true;
+const bool GlobalConfiguration::USE_SYMBOLIC_BOUND_TIGHTENING = false;
```
Comment (guykatzz) [inline src/configuration/GlobalConfiguration.cpp]: Do these changes belong in this PR?

File: src/configuration/GlobalConfiguration.cpp [hunk 2]
```diff
@@ -45,12 +45,12 @@ const double GlobalConfiguration::COST_FUNCTION_ERROR_THRESHOLD = 0.0000000001;
...
 const bool GlobalConfiguration::USE_LINEAR_CONCRETIZATION = true;
 const double GlobalConfiguration::SYMBOLIC_TIGHTENING_ROUNDING_CONSTANT = 0.00000005;
 
 const bool GlobalConfiguration::PREPROCESS_INPUT_QUERY = true;
-const bool GlobalConfiguration::PREPROCESSOR_ELIMINATE_VARIABLES = true;
+const bool GlobalConfiguration::PREPROCESSOR_ELIMINATE_VARIABLES = false;
```
Comment (ahmed-irfan) [inline src/configuration/GlobalConfiguration.cpp]: ditto as pointed out by @guykatzz

File: src/input_parsers/AcasNnet.cpp [hunk 3]
```diff
@@ -32,7 +32,7 @@ AcasNnet *load_network(const char* filename)
     }
 
     //Initialize variables
-    int bufferSize = 10240;
+    int bufferSize = 20480;
```
Comment (ahmed-irfan) [inline src/input_parsers/AcasNnet.cpp]: i guess it is not for this PR

File: src/configuration/GlobalConfiguration.cpp (hunk 1 above)
Comment (anwu1219) [reply src/configuration/GlobalConfiguration.cpp]: Oops, I reverted this change.

**Pull request merged**